monai/inferers/merger.py

```diff
@@ -170,6 +186,177 @@ def get_counts(self) -> torch.Tensor:
         Get the aggregator tensor for number of samples.
 
         Returns:
-            torch.Tensor: Number of accumulated samples at each location.
+            torch.Tensor: number of accumulated samples at each location.
+        """
+        return self.counts
+
+
+@require_pkg(pkg_name="zarr")
+class ZarrAvgMerger(Merger):
+    """Merge patches by taking average of the overlapping area and store the results in zarr array.
```

Comment:
I think you really need to explain here what zarr is, it's the first that comes to mind reading this. It would be helpful to explain what it's used for and why.

Reply:
Sure thing!

Reply:
Added more info about zarr in the docstring.

Reply:
In response to @ericspod's question about where else Zarr might apply: it could have a role in patch-based training pipelines. i.e. the first time, you load the patch directly from the study/WSI into Zarr. Thereafter, you just use the Zarr array.